### ISSUE TITLE
ofLog::setAutoSpace() does nothing

### ISSUE THREAD

**Issue by armadillu** [openframeworks/openFrameworks#4419]
```
ofLogNotice("myModule") << "hello" 
```
gets me:

```
[notice ] myModule: hello
```

no matter what ```ofLog::setAutoSpace()``` is set to (true or false);

I was expecting to get
```
[notice ] myModule:hello
```

when ```ofLog::setAutoSpace(false)```is called; but not entirely sure what it's supposed to do.

I checked [ofLog.cpp](https://github.com/openframeworks/openFrameworks/blob/ec9f59f65706ca35138626a44e71fab6f778354a/libs/openFrameworks/utils/ofLog.cpp#L81) and it basically sets a static string variable named ```padding``` to "" or " "; but then its used nowhere.

**Comment by bilderbuchi** [comment]
this was used to set the padding between consecutive `<<` uses (but probably not the first one). It's used in ofLog.h [here](https://github.com/openframeworks/openFrameworks/blob/ec9f59f65706ca35138626a44e71fab6f778354a/libs/openFrameworks/utils/ofLog.h#L433), cf. #761. Pinging @danomatika

**Comment by danomatika** [comment]
Christoph is right. Try it with the following ...

    ofLogNotice("myModule") << "hello" << "world" << 1.234;

**Comment by arturoc** [comment]
i'm going to close this since i think it's just a missunderstanding of how the function works. for a while when we introduced ofLog we had it automatically add spaces between every << so you didn't need to do:

```
ofLogNotice("myModule") << "hello" << " " << 1.234;
```
and just doing:

```
ofLogNotice("myModule") << "hello" << 1.234;
```

would add the space between "hello" and 1.234. at some point it was decided that that wasn't very standard so we decided to keep it as an option for whoever wanted to use it.
